admin/views/licenses-legacy.php

```diff
@@ -213,8 +217,7 @@
 								__( 'More information %1$sabout %3$s%2$s', 'wordpress-seo' ),
 								'<span class="screen-reader-text">',
 								'</span>',
-								$extension->title
-							);
+								$extension->title );
```

Comment:
Confirmed as the same issue (with the same solution) as https://github.com/Yoast/wordpress-seo/pull/6875#discussion_r133187343